src/app/node-data/component.ts

```diff
@@ -341,7 +327,7 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
         const defaultTemplate = this._datacenterSpec.spec.vsphere.templates
```

Comment:
I'd simply return directly here as there is no additional check now.

Reply:
👍